frontend/src/lib/components/LemonCheckbox/LemonCheckbox.tsx

```diff
@@ -75,7 +77,7 @@ export function LemonCheckbox({
                         checked={localChecked}
                         defaultChecked={defaultChecked}
                         onChange={(e) => {
-                            setLocalChecked(e.target.checked)
+                            checked === undefined && setLocalChecked(e.target.checked)
```

Comment:
Hmm... This is kind of hard to follow. I guess instead of here, `localChecked` will get set in `useEffect` for the case of a controlled input? Might be good to explicitly specify this with a comment here.

Reply:
Yeah exactly. Without this it creates a double-click effect. I'll add a comment